ccl/changefeedccl/cdceval: TestEvaluator failed

ccl/changefeedccl/cdceval.TestEvaluator [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11229264?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11229264?buildTab=artifacts#/) on master @ [1382b26a97bf6f70a07d363dc319283c173359eb](https://github.com/cockroachdb/cockroach/commits/1382b26a97bf6f70a07d363dc319283c173359eb):


```
=== RUN   TestEvaluator
    test_log_scope.go:167: test logs captured to: /artifacts/tmp/_tmp/4f079cd13f8a776a99f45e473cd32df8/logTestEvaluator4214666620
    test_log_scope.go:81: use -show-logs to present logs inline
=== CONT  TestEvaluator
    expr_eval_test.go:628: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4f079cd13f8a776a99f45e473cd32df8/logTestEvaluator4214666620
--- FAIL: TestEvaluator (16.07s)
=== RUN   TestEvaluator/user_defined_function
    expr_eval_test.go:573: timeout reading row
    --- FAIL: TestEvaluator/user_defined_function (5.04s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEvaluator.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-30434

Epic CRDB-11732

--- a/pkg/ccl/changefeedccl/cdctest/row.go
+++ b/pkg/ccl/changefeedccl/cdctest/row.go
@@ -53,7 +53,7 @@ func MakeRangeFeedValueReader(
 	)
 	require.NoError(t, err)
 
-	var timeout = 5 * time.Second
+	var timeout = 10 * time.Second
 	if util.RaceEnabled {
 		timeout = 3 * timeout
 	}

